react-native-libraries.json

```
@@ -13470,5 +13470,15 @@
     "tvos": true,
     "expoGo": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/gronxb/hot-updater/tree/main/packages/react-native",
+    "npmPkg": "@hot-updater/react-native",
+    "images": [
+      "https://github.com/gronxb/hot-updater/blob/main/demo.gif?raw=true"
```

Hey @gronxb, thanks for adding a new entry to the directory! 👍 

Just a small nit, let's use already resolved version of raw URL:
```suggestion
      "https://raw.githubusercontent.com/gronxb/hot-updater/refs/heads/main/demo.gif"
```

---

Thanks ! Done